src/App.js

```diff
@@ -1,24 +1,58 @@
-import logo from './logo.svg';
+import { getDatabase, ref, onValue } from 'firebase/database';
+import { useEffect, useState } from 'react';
+import firebase from './firebase';
+import Header from './Header';
+import Form from './Form';
+import ListContainer from './ListContainer';
 import './App.css';
 
 function App() {
+  // setting useState for input data when the page is mounted
+  const [ userInputs, setUserInputs ] = useState([]);
+
+  useEffect( () => {
+    const database = getDatabase(firebase);
+    const dbRef = ref(database);
+    // onValue event listener that will run the code upon the changes on database value
+    onValue(dbRef, (res) => {
+      // get userInput data values from database
+      const data = res.val();
+      // accessing each data in object format and turn them into array
+      const newState = [];
+      for (let userInput in data) {
+        const dataObject = {
+          id: userInput,
+          day: data[userInput]['day'],
+          event: data[userInput]['event'],
+          detail: data[userInput]['detail'],
+        };
+        newState.push(dataObject);
+      };
+      setUserInputs(newState);
+    });
+  }, [] );
+
+  // reset button for the APP
+  const reset = () => {
+    setUserInputs([]);
+  }
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <div className='wrapper app__container'>
+        <header>
+          <Header />
+        </header>
```

Comment:
What do you think about pulling the `<header>` tags into the `<Header />` component itself?